contrib/opentimelineio_contrib/adapters/kdenlive.py

```diff
@@ -164,7 +164,10 @@ def clock(time):
 
 def write_keyframes(kfdict):
     """Build a MLT keyframe string"""
-    return ';'.join('{}={}'.format(str(int(t.value))).format(v)
+    # TODO: The following line is causing a lint error:
```

Comment:
Is there any reason not to unpack this line a little? There needs to be a one to one mapping of braces to format arguments, and lint is correct. The join on semicolon is also suspicious, since there is only one string involved. Is the line intending:

```py
return '{}={}'.format(str(int(t.value)), v) + ';'
```
?

Reply:
There is more than one string, there is an iteration over the `.items()` of the `kfdict`, and I think this is fine.  (contrib area)

Reply:
Yeah, I agree with you @meshula that this should ideally have a real code fix.
Since this requires attention from the adapter maintainer and is in the contrib area, I figured it's best to just silence the lint error for now to get the code building again. I've filed #703 so this issue doesn't totally disappear.